Signal/src/Profiles/OWSProfileManager.m

```diff
@@ -1384,14 +1384,29 @@ - (nullable UIImage *)loadProfileAvatarWithFilename:(NSString *)filename
     return image;
 }
 
++ (NSString *)oldProfileAvatarsDirPath
+{
+    return [[OWSFileSystem appDocumentDirectoryPath] stringByAppendingPathComponent:@"ProfileAvatars"];
+}
+
++ (NSString *)newProfileAvatarsDirPath
```

Comment:
nit: what's old will always be old, but nothing is new forever, and calling it "new" outside of the context of the migration is not clarifying.

suggested name:
`newProfileAvatarsDirPath` -> `profileAvatarsDirPath`

And maybe a bit clearer:
`oldProfileAvatarsDirPath` -> `profileAvatarsOldDirPath` or `profileAvatarsLegacyDirPath`

I feel the same about the similar names in TSAttachmentStream and TSStorageManager

Reply:
I can't do: `newProfileAvatarsDirPath` -> `profileAvatarsDirPath` because that would conflict with the `once` getter which does a bunch of work.  

Instead, I'm replacing `old` and `new` with `legacy` and `sharedData` here and everywhere else we're doing a similar migration.